package.json

```diff
@@ -4,10 +4,17 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npm run build && cd .. & node index.js",
-    "startProd": "node index.js",
+    "startDev": "nodemon",
+    "build_client": "cd client &&  elm make src/Main.elm --output=../public/index.js && cd ..",
+    "start": "node index.js",
```

Comment:
what do you think about adding `--inspect` here so we can debug in the browser? https://nodejs.org/en/docs/guides/debugging-getting-started/

Reply:
sure let's add it.